### DISCUSSION TITLE
Doris be节点宕机 be版本 1.2.0

### DISCUSSION THREAD

**Discussion by qiaoshiling** [apache/doris#21868] [A - General / Q&A]
*** Query id: 0-0 ***
*** Aborted at 1689561317 (unix time) try "date -d @1689561317" if you are using GNU date ***
*** Current BE git commitID: Unknown ***
*** SIGSEGV address not mapped to object (@0x7fa0b8bf7000) received by PID 13087 (TID 0x7fa0ea3ff700) from PID 18446744072514138112; stack trace: ***
 0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /root/doris/be/src/common/signal_handler.h:420
 1# os::Linux::chained_handler(int, siginfo*, void*) in /usr/local/jdk1.8.0_102/jre/lib/amd64/server/libjvm.so
 2# JVM_handle_linux_signal in /usr/local/jdk1.8.0_102/jre/lib/amd64/server/libjvm.so
 3# signalHandler(int, siginfo*, void*) in /usr/local/jdk1.8.0_102/jre/lib/amd64/server/libjvm.so
 4# 0x00007FA151456400 in /lib64/libc.so.6
 5# phmap::priv::raw_hash_set<phmap::priv::FlatHashSetPolicy<doris::StringValue>, phmap::Hash<doris::StringValue>, phmap::EqualTo<doris::StringValue>, std::allocator<doris::StringValue> >::iterator phmap::priv::raw_hash_set<phmap::priv::FlatHashSetPolicy<doris::StringValue>, phmap::Hash<doris::StringValue>, phmap::EqualTo<doris::StringValue>, std::allocator<doris::StringValue> >::find<doris::StringValue>(doris::StringValue const&) at /var/local/thirdparty/installed/include/parallel_hashmap/phmap.h:1647
 6# doris::InListPredicateBase<(doris::PrimitiveType)23, (doris::PredicateType)7>::evaluate(doris::vectorized::IColumn const&, unsigned short*, unsigned short) const at /root/doris/be/src/olap/in_list_predicate.h:244
 7# doris::segment_v2::SegmentIterator::_evaluate_short_circuit_predicate(unsigned short*, unsigned short) at /root/doris/be/src/olap/rowset/segment_v2/segment_iterator.cpp:1073
 8# doris::segment_v2::SegmentIterator::next_batch(doris::vectorized::Block*) at /root/doris/be/src/olap/rowset/segment_v2/segment_iterator.cpp:1177
 9# doris::BetaRowsetReader::next_block(doris::vectorized::Block*) at /root/doris/be/src/olap/rowset/beta_rowset_reader.cpp:278
10# doris::vectorized::VCollectIterator::Level0Iterator::next(doris::vectorized::Block*) at /root/doris/be/src/vec/olap/vcollect_iterator.cpp:268
11# doris::vectorized::VCollectIterator::Level1Iterator::_normal_next(doris::vectorized::Block*) at /root/doris/be/src/vec/olap/vcollect_iterator.cpp:534
12# doris::vectorized::VCollectIterator::Level1Iterator::_normal_next(doris::vectorized::Block*) at /root/doris/be/src/vec/olap/vcollect_iterator.cpp:542
13# doris::vectorized::VCollectIterator::Level1Iterator::next(doris::vectorized::Block*) at /root/doris/be/src/vec/olap/vcollect_iterator.cpp:361
14# doris::vectorized::VCollectIterator::next(doris::vectorized::Block*) at /root/doris/be/src/vec/olap/vcollect_iterator.cpp:191
15# doris::vectorized::BlockReader::_direct_next_block(doris::vectorized::Block*, doris::MemPool*, doris::ObjectPool*, bool*) at /root/doris/be/src/vec/olap/block_reader.cpp:168
16# doris::vectorized::BlockReader::next_block_with_aggregation(doris::vectorized::Block*, doris::MemPool*, doris::ObjectPool*, bool*) at /root/doris/be/src/vec/olap/block_reader.h:46
17# doris::vectorized::NewOlapScanner::_get_block_impl(doris::RuntimeState*, doris::vectorized::Block*, bool*) at /root/doris/be/src/vec/exec/scan/new_olap_scanner.cpp:318
18# doris::vectorized::VScanner::get_block(doris::RuntimeState*, doris::vectorized::Block*, bool*) at /root/doris/be/src/vec/exec/scan/vscanner.cpp:53
19# doris::vectorized::ScannerScheduler::_scanner_scan(doris::vectorized::ScannerScheduler*, doris::vectorized::ScannerContext*, doris::vectorized::VScanner*) at /root/doris/be/src/vec/exec/scan/scanner_scheduler.cpp:236
20# doris::ThreadPool::dispatch_thread() at /root/doris/be/src/util/threadpool.cpp:542
21# doris::Thread::supervise_thread(void*) at /root/doris/be/src/util/thread.cpp:455
22# start_thread in /lib64/libpthread.so.0
23# clone in /lib64/libc.so.6

**Comment by LemonLiTree** [comment]
Upgrade to 1.2.6, if be is still down, please post the log of be.out
